release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3061,3 +3061,24 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.14.0 --pullspecs
 ----
 //replace 4.y.z for the correct values for the release. You do not need to update oc to run this command.
+
+[id="ocp-4-14-1-ga"]
+=== RHBA-2023:6153 - {product-title} 4.14.1 bug fix update
+
+Issued: 2023-11-01
+
+{product-title} release {product-version}.0, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:6153[RHBA-2023:6153] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:6152[RHSA-2023:6152] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.1 --pullspecs
+----
+
+[id="ocp-4-14-1-updating"]
+==== Updating
+
+To update an existing {product-title} 4.14 cluster to this latest release, see xref: https://docs.openshift.com/container-platform/4.14/updating/updating_a_cluster/updating-cluster-cli.html[Updating a cluster using the CLI].
```

Comment:
This link appears a bit mixed up. `xref` is there, and showing as plain text in the preview, but the link is written like an external link with `https:`.